src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimDataContentProvider.java

```diff
@@ -68,8 +69,10 @@ public ClaimContent createContent(Claim claim) {
         List<EvidenceContent> evidences = null;
         Optional<Evidence> evidence = claim.getClaimData().getEvidence();
         if (evidence.isPresent()) {
-            evidences = evidence.get().getRows()
+            evidences = Optional.ofNullable(evidence.get().getRows())
```

Comment:
why would evidence be present but not the rows?

Reply:
front-end is sending in case of no evidence was entered.
```
	"evidence": {
		"rows": []
	},
```

Reply:
that's an empty list then? and should be fine?

Reply:
Tim is correct -- please investigate the problem more throughly so that we know why this issue is happening.

Reply:
Can we please investigate that more, we should avoid coding against empty lists.

Also I would say evidence object should be submitted only when list is populated. Can we fix frontend conversion and put some validation against empty list of rows?

Reply:
When this is mapped to Model object Jackson converts to null list/array if empty.

Reply:
This is what we get saved in claim-store database,
```
 "evidence": {},
```

Reply:
that doesn't seem right, fix might be be to stop sending that, a data migration to delete any of those should be safe too

Reply:
I would be happy to accept current solution as technical debt however I had a chat with @sabahirfan and he is happy to make changes to:

1. send evidence object only when at least one row is populated
2. validate evidence object on backend side to make sure if submitted there is at least one row
3. check whether we have any of evidence data in production DB and migrate if necessary

Reply:
On point 2. There is one concern. We will have different validation logics for front-end and back-end.